sim/src/tlv.rs

```diff
@@ -385,9 +404,14 @@ impl ManifestGen for TlvGen {
             estimate += 4 + if aes256 { 96 } else { 80 };
         }
 
-        // Gather the size of the dependency information.
+        // Gather the size of the dependency information and security counters.
```

Comment:
Let's make it generic.
```suggestion
        // Gather the size of the protected TLV area.
```